examples/NewModuleTemplate/build.psake.ps1

```diff
@@ -70,8 +70,8 @@ Properties {
 
     # The root directory of the module source and tests.  It could be the workspace
     # root or a subdir such as src, module, <my-module-name>.
-    $SourceRootDir = "$PSScriptRoot/src"
-    $TestRootDir   = "$PSScriptRoot/test"
+    $SourceRootDir = Join-Path -Path $PSScriptRoot -ChildPath "src"
+    $TestRootDir   = Join-Path -Path $PSScriptRoot -ChildPath "Tests"
 
```

Comment:
Same as above except that the test root dir should be `test`.  We recently decided on a standard directory naming convention.  The template has not been updated yet but it will be.